extruct/rdfa.py

```diff
@@ -51,4 +153,10 @@ def extract_items(self, document, base_url=None, expanded=True):
                           check_lite=False)
         g = PyRdfa(options, base=base_url).graph_from_DOM(document, graph=Graph(), pgraph=Graph())
         jsonld_string = g.serialize(format='json-ld', auto_compact=not expanded).decode('utf-8')
-        return json.loads(jsonld_string)
\ No newline at end of file
+        
+        try:
+            # hack to fix the ordering of multi-value properties (see issue 116)
+            # it should be disabled once PyRDFA fixes itself
+            return self._fix_order(jsonld_string, document)
+        except:
```

Comment:
Why all exceptions are ignored here? Is it possible to be more explicit with errors?

Reply:
I'll answer that one. The idea here is that `_fix_order` is just a temporal hack to be removed once `rdflib` fixes the ordering issue somehow. So the idea was that this code should never harm original implementation. Here is where it is ensured: In case any exception is raised by `_fix_order`, the original implementation should be invoked as a fallback. There are also some unit tests that check for that.

Reply:
I see, thanks for clarification @ivanprado . Good that we also have tests which make sure that the re-ordering works.